config/master/aws-k8s-cni.yaml

```diff
@@ -97,20 +97,46 @@
                 - "fargate"
       "containers":
       - "env":
+        - "name": "ADDITIONAL_ENI_TAGS"
+          "value": "{}"
+        - "name": "AWS_VPC_CNI_NODE_PORT_SUPPORT"
+          "value": "true"
         - "name": "AWS_VPC_ENI_MTU"
           "value": "9001"
         - "name": "AWS_VPC_K8S_CNI_CONFIGURE_RPFILTER"
           "value": "false"
+        - "name": "AWS_VPC_K8S_CNI_CUSTOM_NETWORK_CFG"
+          "value": "false"
+        - "name": "AWS_VPC_K8S_CNI_EXTERNALSNAT"
+          "value": "false"
         - "name": "AWS_VPC_K8S_CNI_LOGLEVEL"
           "value": "DEBUG"
         - "name": "AWS_VPC_K8S_CNI_LOG_FILE"
           "value": "/host/var/log/aws-routed-eni/ipamd.log"
+        - "name": "AWS_VPC_K8S_CNI_RANDOMIZESNAT"
+          "value": "prng"
         - "name": "AWS_VPC_K8S_CNI_VETHPREFIX"
           "value": "eni"
+        - "name": "AWS_VPC_K8S_PLUGIN_LOG_FILE"
+          "value": "/var/log/aws-routed-eni/plugin.log"
+        - "name": "AWS_VPC_K8S_PLUGIN_LOG_LEVEL"
+          "value": "DEBUG"
+        - "name": "DISABLE_INTROSPECTION"
+          "value": "false"
+        - "name": "DISABLE_METRICS"
+          "value": "false"
+        - "name": "ENI_CONFIG_ANNOTATION_DEF"
+          "value": "k8s.amazonaws.com/eniConfig"
+        - "name": "ENI_CONFIG_LABEL_DEF"
+          "value": "k8s.amazonaws.com/eniConfig"
+        - "name": "INTROSPECTION_BIND_ADDRESS"
+          "value": "127.0.0.1:61679"
```

Comment:
I think we can actually leave these three out. The ENI_CONFIG ones are only used when custom networking is enabled, and that is off by default. Also, the introspection port is local host only, and used in our debug script. It is enough to show the DISABLE_XXXX flags.

Reply:
Based on #1053, I assumed all variables are required. Alright. I will add a new commit with changes.